### PR TITLE
chore(localdebug): add npm install task

### PR DESCRIPTION
**Pull request by XiaofuHuang**
Detail: https://msazure.visualstudio.com/Microsoft%20Teams%20Extensibility/_workitems/edit/15310382
image

### CODE REVIEW

**Comment by swatDong** [comment]
How could user know error details for install failure?

**Comment by XiaofuHuang** [comment]
> How could user know error details for install failure?

image
image

**Review by swatDong — APPROVED** [review]

**Review by swatDong — APPROVED** [review]

**Review by kuojianlu — DISMISSED** [review]

File: packages/vscode-extension/package.nls.json [hunk 1]
```diff
@@ -283,8 +283,11 @@
     "teamstoolkit.localDebug.portsAlreadyInUse": "Ports: %s are already in use. Close these ports and try again.",
     "teamstoolkit.localDebug.portWarning": "Changing port(s) in package.json may break local debug. Ensure all port changes are expected or refer to documentation by clicking `Learn More` button. (%s package.json location: %s)",
     "teamstoolkit.localDebug.prerequisitesCheckFailure": "Prerequisites Check Failed. Please check the %s to see the details. If you wish to bypass checking and installing any prerequisites, you can disable them in Visual Studio Code settings.",
-    "teamstoolkit.localDebug.outputPanel": "output panel",
+    "teamstoolkit.localDebug.npmInstallFailure": "NPM Package Install Task Failed. Please check the %s to see the details.",
+    "teamstoolkit.localDebug.outputPanel": "[output panel](command:fx-extension.showOutputChannel)",
```
Comment (tecton) [inline packages/vscode-extension/package.nls.json:287]: Please add a comment like `_xxx.comment` to note that command doesn't need localization. Only the output panel is translated.

Comment (XiaofuHuang) [reply packages/vscode-extension/package.nls.json]: Moved the command outof `package.nls.json`

**Review by kuojianlu — APPROVED** [review]

**Review by swatDong — APPROVED** [review]

**Pull request merged**